webapps/camunda-bpm-sdk-js/lib/api-client/resources/process-instance.js

```diff
@@ -209,13 +209,36 @@ var ProcessInstance = AbstractClientResource.extend(
       });
     },
 
+    /**
+     * Sets variables to process instances (batch).
+     *
+     * @see https://docs.camunda.org/manual/develop/reference/rest/process-instance/post-set-variables-async/
```

Comment:
The JS-Docs usually refer to the `latest` branch, e.g. https://github.com/camunda/camunda-bpm-platform/blob/44cceb926f572749d452e16a5c94c93c887162ff/webapps/camunda-bpm-sdk-js/lib/api-client/resources/process-instance.js#L184 

```suggestion
     * @see https://docs.camunda.org/manual/latest/reference/rest/process-instance/post-set-variables-async/
```

Reply:
👍